CefSharp.Core/ManagedCefBrowserAdapter.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Stdafx.h"
```

Comment:
Added license. I only moved the bare minimum to get it to compile since the .h files include each otherwise. I can move the rest though.